plugins/updater/src/updater.rs

```diff
@@ -1026,6 +1054,32 @@ impl PathExt for PathBuf {
     }
 }
 
+#[cfg(windows)]
+fn escape_msi_property_arg(arg: impl AsRef<OsStr>) -> String {
+    let mut arg = arg.as_ref().to_string_lossy().to_string();
+
+    // Otherwise this argument will get lost in ShellExecute
+    if arg.is_empty() {
```

Comment:
if an arg.is_empty, it should be skipped no?

Reply:
Nope. This should be a pretty rare case but for example if the user starts the initial app with `./myapp '""'` arg.is_empty() will be true and really just an empty rust string. If we forward it to ShellExecute we'll and up with literally nothing (no argument) since we're appending an empty string to a string. Therefore we need to wrap the empty string in `"{empty string}"` but since we're inside the msi prop we need it to be `""{empty string}""`. Only then will the new app instance get an empty string in `env::args()` like the initial instance.